### PR TITLE
Fixes for integration test(s) of IoTHubManager.

### PR DESCRIPTION
**Pull request by sushilraje**
# Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] All new and existing tests passed.
- [ ] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change

...

# Motivation for the change

...

### CODE REVIEW

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java [hunk 1]
```diff
@@ -63,10 +63,12 @@ public static Configuration toHubConfiguration(final DeploymentServiceModel mode
 
         // Required labels
         labels.put(PACKAGE_TYPE_LABEL, model.getPackageType().toString());
```
Comment (ppathan) [inline iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java]: Should we check for empty and null values in other labels?

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java [hunk 2]
```diff
@@ -63,10 +63,12 @@ public static Configuration toHubConfiguration(final DeploymentServiceModel mode
...
-        labels.put(CONFIG_TYPE_LABEL, model.getConfigType().toString());
         labels.put(DEPLOYMENT_NAME_LABEL, model.getName());
         labels.put(DEPLOYMENT_GROUP_ID_LABEL, deploymentGroup.getId());
         labels.put(RM_CREATED_LABEL, Boolean.TRUE.toString());
+        if (!StringUtils.isBlank(model.getConfigType().toString())){
+            labels.put(CONFIG_TYPE_LABEL, model.getConfigType().toString());
+        }
```
Comment (ppathan) [inline iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java]: Can you put empty in else block?

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/models/DeploymentServiceModel.java [hunk 3]
```diff
@@ -84,7 +84,13 @@ public DeploymentServiceModel(Configuration deployment) throws
             this.packageType = PackageType.deviceConfiguration;
         }
 
-        this.configType = deployment.getLabels().get(ConfigurationsHelper.CONFIG_TYPE_LABEL.toString());
+        if (deployment.getLabels().containsKey(ConfigurationsHelper.CONFIG_TYPE_LABEL.toString())) {
```
Comment (ppathan) [inline iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/models/DeploymentServiceModel.java]: If you can empty in the else block then this file doesn't need to change, i think.

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java (hunk 1 above)
Comment (sushilraje) [reply iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java]: As per our discussion, we are checking in the service/controller layer, this is a helper function, so I don't think we need this.

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java (hunk 2 above)
Comment (sushilraje) [reply iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java]: Empty string causes exception, so we has to add the if/else block.

**Pull request merged**